src/main/java/Duke.java

```diff
@@ -1,10 +1,221 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
+
 public class Duke {
+
+    private static final String filePath = "data/duke.txt";
+
+    /*private static void writeToFile(String filePath, String textToAdd) throws IOException {
+        FileWriter fw = new FileWriter(filePath);
+        fw.write(textToAdd);
+        fw.close();
+    }*/
+
+    private static void appendToFile(String filePath, String textToAppend) throws IOException {
+        FileWriter fw = new FileWriter(filePath, true); // create a FileWriter in append mode
+        fw.write(textToAppend);
+        fw.close();
+    }
+
     public static void main(String[] args) {
+
+        File f = new File(filePath);
+        File folder = new File("data");
+        if (!folder.exists()) {
+            folder.mkdir();
+        }
+        if (!f.exists()) {
+            try {
+                f.createNewFile();
+            } catch (IOException e) {
+                System.out.println("Failed to create file to save tasks.");
+            }
+        }
+
+        System.out.println("full path: " + f.getAbsolutePath());
         String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+                    + "|  _ \\ _   _| | _____ \n"
+                    + "| | | | | | | |/ / _ \\\n"
+                    + "| |_| | |_| |   <  __/\n"
+                    + "|____/ \\__,_|_|\\_\\___|\n";
+        //System.out.println("Hello from\n" + logo);
+        String GREET = "Hello! I'm Duke\n" + "What can I do for you?\n";
+        String EXIT = "Bye. Hope to see you again soon!\n";
+        String BYE = "bye";
+        String LIST = "list";
+        String MARK = "mark";
+        String TODO = "todo";
+        String DEADLINE = "deadline";
+        String EVENT = "event";
+        String DELETE = "delete";
```

Comment:
You might want to add a prefix to these constants if they have similar purpose or belongs in the same "group", if that makes sense. For instance, `COMMAND_BYE`, `COMMAND_LIST`, etc. This can improve the clarity and readability of the code.